NickvisionTagger.GNOME/Views/MainWindow.cs

```diff
@@ -1230,7 +1239,15 @@ private void KeyboardShortcuts(Gio.SimpleAction sender, EventArgs e)
     /// </summary>
     /// <param name="sender">Gio.SimpleAction</param>
     /// <param name="e">EventArgs</param>
-    private void Quit(Gio.SimpleAction sender, EventArgs e) => _application.Quit();
+    private void Quit(Gio.SimpleAction sender, EventArgs e)
+    {
+        GetDefaultSize(out var width, out var height);
+        _controller.WindowWidth = width;
+        _controller.WindowHeight = height;
+        _controller.WindowMaximized = IsMaximized();
+        Aura.Active.SaveConfig("config");
+        _application.Quit();
```

Comment:
I just realized, shouldn't this call `Close()` that way if there are unapplied changes it asks the user that to do with them?

In that case it would become:
```
private void Quit(Gio.SimpleAction sender, EventArgs e) => Close();
```
as the window size code is handled inside the close requested event.

Reply:
`OnCloseRequest` is called when the user clicks on the close button, not when we call `Close` from code.

Reply:
So then we could make it:
```csharp
private void Quit(Gio.SimpleAction sender, EventArgs e) => OnCloseRequested(this, e);
```